src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  min-height: 300px;
```

Comment:
```suggestion
  height: 300px;
```
I guess it is just a height
![image](https://user-images.githubusercontent.com/96409563/225259645-95f0be21-22ea-48d1-869d-0da93d5e783b.png)